src/redprl/syntax/operator.sig

```diff
@@ -36,6 +36,16 @@ struct
    | PUSHOUT | LEFT | RIGHT | GLUE | PUSHOUT_REC
    (* coequalizer *)
    | COEQUALIZER | CECOD | CEDOM | COEQUALIZER_REC
+
+   (* inductive types *)
+   | IND_RECTYPE_SELF | IND_RECTYPE_FUN
+   | IND_RECTERM_INTRO of opid
+   | IND_RECTERM_FCOM | IND_RECTERM_LAM | IND_RECTERM_APP
+   | IND_CONSTRUCTOR of {label : opid, ndim : int, nvar : int, nrecvar : int}
+   | IND_ELIM_MK_CASE of {label : opid, ndim : int, nvar : int, nrecvar : int}
+   | IND_INTRO of {indtype : opid, label : opid}
+   | IND_ELIM of {iondtype : opid}
```

Comment:
Did you mean `indtype` here?

Reply:
Yes. I was forcing SML/NJ to recompile and then...